art/estimators/object_detection/PyTorchFasterRCNN.py

```diff
@@ -184,14 +206,13 @@ def predict(self, x, **kwargs):
 
         # Apply preprocessing
         x, _ = self._apply_preprocessing(x, y=None, fit=False)
-        x = x.astype(np.uint8)
 
         transform = torchvision.transforms.Compose([torchvision.transforms.ToTensor()])
 
         image_tensor_list = list()
 
         for i in range(x.shape[0]):
-            image_tensor_list.append(transform(x[i]))
+            image_tensor_list.append(transform(x[i] / self.clip_values[1]).to(self._device))
```

Comment:
See the previous comment.

Reply:
You are right. The reason for this is that this estimator is model-specific for PyTorch's FasterRCNN which expects images in the pixel range [0, 1]. To enable inputs to `PyTorchFasterRCNN` in the range [0, 255] it normalises with `clip_values[1]`. I have now added assertions to prevent unexpected `clip_values` where where `min_clip!=0`.